content/docs/capabilities/self-hosted-authenticate-service.md

```diff
@@ -3,81 +3,60 @@ id: self-hosted-authenticate-service
 keywords: [self-hosted authenticate service url, self-hosted identity provider]
-description: Use Pomerium's Self-Hosted Authenticate Service to set up and deploy Pomerium with your own hosted settings.
+description: Use Pomerium's Self-Hosted Authenticate Service to deploy Pomerium with your own identity provider.
 ---
 
 # Self-Hosted Authenticate Service
 
-Pomerium's **Self-Hosted Authenticate Service** allows you to configure your own **Authenticate Service URL** and **Identity Provider**.
+Pomerium's **Self-Hosted Authenticate Service** allows you to configure Pomerium with the identity provider of your choice.
 
-The Self-Hosted Authenticate Service is available for both open-source Pomerium Core and Enterprise users.
+The self-hosted authenticate service is available for open-source Pomerium, Pomerium Zero, and Pomerium Enterprise.
 
-## How the Self-Hosted Authenticate Service works
+:::info
 
-Self-hosting your Pomerium instance requires you to configure an [Authenticate Service URL](/docs/reference/service-urls#authenticate-service-url) and an OIDC-compliant [identity provider](/docs/identity-providers) (IdP).
+The _authenticate service_ is one of the [four logical services](/docs/internals/architecture#component-level) that make up Pomerium. The authenticate service is unique in that Pomerium can either run this service itself, or instead use the [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service).
+
+:::
+
+## How the self-hosted authenticate service works
+
+To configure the self-hosted authenticate service, you will need to set up:
+
+- a URL for the authenticate service
+- an identity provider (IdP)
 
 ### Authenticate service URL
 
-The authenticate service URL is an externally accessible URL used by Pomerium's Authenticate Service. You can include your own URL for testing or use Pomerium's localhost URL, which is hardcoded to point to `127.0.0.1`.
+The authenticate service URL is a public-facing URL that should resolve to your Pomerium instance. (If running in split service mode, this URL should resolve to the authenticate service.) Whenever a user is required to log in to access a route, Pomerium will redirect the user to the authenticate service using this URL.
 
-```yaml
-authenticate_service_url: https://authenticate.localhost.pomerium.io
-```
+This URL should not contain a path or query parameters. For example, `https://authenticate.corp.example.com`.
 
-### Identity provider
+This URL must be different from any of the route URLs you use with Pomerium.
 
-Pomerium's Authenticate Service requires an IdP to authenticate and authorize users. Pomerium supports all major IdP solutions and any IdP that uses OAuth 2.0 and OIDC protocols as well.
+See the [Authenticate Service URL](/docs/reference/service-urls#authenticate-service-url) reference page for details on how to configure this setting.
 
-## Configure the Self-Hosted Authenticate Service
+:::info Why does Pomerium use a separate URL for this?
```

Comment:
```suggestion
:::info Why does Pomerium use a separate authenticate service URL?
```

Reply:
@kenjenkins appreciate the edit. I tweaked the admonition title slightly but otherwise, I think this is a decent improvement. If you have no other suggestions, I'll merge.

Reply:
Sure, sounds good.